loader/windows/icd_windows_hkr.h

```diff
@@ -17,5 +17,13 @@
  */
 
 #include <stdbool.h>
+#include <windows.h>
 
 bool khrIcdOsVendorsEnumerateHKR(void);
+
+extern const LUID ZeroLuid;
+
+BOOL adapterAdd(const char* szName, LUID luid);
```

Comment:
We have inconsistency with the naming, either camelCase or PascalCase: getOpenCLRegKeyName() / AdapterAdd / AddAdapter(const char*...)